server/service/src/sync/program_schemas/report_ltfu.json

```diff
@@ -0,0 +1,28 @@
+{
+  "index": {
+    "template": "template.html",
+    "header": null,
+    "footer": null,
+    "query": "query.graphql"
+  },
+  "entries": {
+    "query.graphql": {
+      "type": "GraphGLQuery",
+      "data": {
+        "query": "query LTFU($storeId: String, $now: String) {\n  store(id: $storeId) {\n    ... on StoreNode {\n      storeName\n    }\n  }\n\n  patients(storeId: $storeId) {\n    ... on PatientConnector {\n      nodes {\n        firstName\n        lastName\n        gender\n        phone\n        programEnrolments(filter: { program: { equalTo: \"HIVCareProgram\" } }) {\n          programPatientId\n          events(at: $now, filter: { type: { equalTo: \"programStatus\" } }) {\n            type\n            data\n          }\n          encounters(\n            page: { first: 1, offset: 0 }\n            sort: { key: startDatetime, desc: true }\n            filter: {\n              program: { equalTo: \"HIVCareProgram\" }\n              status: { equalTo: \"COMPLETED\" }\n              startDatetime: { beforeOrEqualTo: $now }\n            }\n          ) {\n            nodes {\n              events(\n                at: $now\n                filter: { type: { equalTo: \"arvMedicationEndOfSupply\" } }\n              ) {\n                type\n                data\n              }\n\n              startDatetime\n            }\n          }\n\n          nextScheduledEncounters: encounters(\n            page: { first: 1, offset: 0 }\n            sort: { key: startDatetime, desc: false }\n            filter: {\n              program: { equalTo: \"HIVCareProgram\" }\n              status: { equalTo: \"SCHEDULED\" }\n              startDatetime: { afterOrEqualTo: $now }\n            }\n          ) {\n            nodes {\n              startDatetime\n            }\n          }\n        }\n      }\n    }\n  }\n}\n",
```

Comment:
same as the viral load report - the `for` says to me that I should expect the patient name

Reply:
Agree thought the same, think it should mean "for store". Will try to improve when doing some styling